src/core/src/package_managers/YumPackageManager.py

```diff
@@ -176,6 +179,17 @@ def get_other_updates(self):
         self.composite_logger.log("Discovered " + str(len(other_packages)) + " 'other' package entries.")
         return other_packages, other_package_versions
 
+    def __is_image_rhel8_higher_skip_security_plugin(self):
+        """Check image RHEL8+ to disable install yum-plugin-security """
+        if self.env_layer.platform.linux_distribution() is not None:
+            os_offer, os_version, os_code = self.env_layer.platform.linux_distribution();
+
+            if "Red Hat Enterprise Linux" in os_offer and int(os_version.split('.')[0]) >= 8:
+                self.composite_logger.log_debug("Disable RHEL8+ install yum-plugin-security: Os Version " + str(os_version))
+                return True
+
+        return False
```

Comment:
Single responsibility - the _only_ responsibility of the code here is to know if it's RHEL 8 or greater.
The name of the function should reflect that. The actions inside the function should reflect that.
There should be no reference to what the calling code wants or will do with the return.

Reply:
updated